add-workers/README.md

```diff
@@ -0,0 +1,7 @@
+# Add workers
+
+This folder is where the configuration to add day2 worker nodes is stored.
+
+To add a day2 worker node, one needs to add a directory with some yaml in the `add-workers` directory. See the [overlay](https://github.com/redhat-partner-solutions/vse-ocp-bases/tree/main/overlay) directory in the `bases` repo for examples.
```

Comment:
Could with being a bit more descriptive than `some yaml` perhaps some thing like `the specific worker configuration`?